github/repos_commits.go

```diff
@@ -241,6 +240,34 @@ func (s *RepositoriesService) CompareCommits(ctx context.Context, owner, repo st
 	return comp, resp, nil
 }
 
+// CompareCommitsRaw compares a range of commits with each other in raw (diff or patch) format.
+//
```

Comment:
Can you please add the following comments from the official docs?

```suggestion
//
// Both "base" and "head" must be branch names in "repo".
// To compare branches across other repositories in the same network as "repo",
// use the format "<USERNAME>:branch".
//
```